### ISSUE TITLE
Incorrect info in guide?

### ISSUE THREAD

**Issue by LoopeyTheGreat** [home-assistant/home-assistant.github.io#36621]
### Feedback

Following the steps in the guide, during linking I was getting a generic "cannot connect at this time" message. Found the solution was that I needed to add ":443" to my web authorization and token URI. Suggest changing the instruction:

_Input all information required. Assuming your Home Assistant can be accessed by https://[YOUR HOME ASSISTANT URL][:PORT], where PORT is the TCP port. **The port can be omitted if it is 443.** For Alexa account linking, by default, the standard port 443 is used by default. Use your firewall to forward this, if needed:_

By removing the statement that the port can be omitted if your port is 443. U

### URL

https://www.home-assistant.io/integrations/alexa.smart_home/

### Version

2024.12.5

**Comment by jbouwh** [comment]
So you say we should always add the port, even when port 443 is used?

**Comment by jbouwh** [comment]
I checked my settings for account linking, but I did not need to add the port:

![afbeelding](https://github.com/user-attachments/assets/59bea27a-84ad-4a82-a68a-bdf3f8f8dece)

**Comment by LoopeyTheGreat** [comment]
The AWS side of all this is admittedly quite a bit over my head.... but my suspicion is that this part of the process is only used during the initial linking of your Alexa account to the Lambd function. I was very meticulous in following the guide, and everything was going well until it came to the linking part. I spent the better part of my day trying to figure out what was going on until I tried adding the port to the address. I got to the conclusion because I was not seeing anything in the Lambd logs showing an attempted connection, so I turned my attention to Firewall/DDNS/address areas. In my troubleshooting I made only one change  at a time before re-attempting the linking, and had success after adding the port.

**Comment by jbouwh** [comment]
Right, but it does not seem logical that this worked without the added port (which was the case) and later would require a port added. So for now I suggest we leave the docs as they are.
